Build/Common.Build.settings

```diff
@@ -98,23 +98,9 @@
 
     <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
 
-    <DevEnvDir Condition="'$(DevEnvDir)' == ''">$([MSBuild]::GetRegistryValueFromView('HKEY_LOCAL_MACHINE\Software\Microsoft\VisualStudio\$(VSTarget)', 'InstallDir', null, RegistryView.Registry32))\</DevEnvDir> 
```

Comment:
This doesn't work for VS 2017, so we're using the $(VSInstallRoot) environment variable which is set.